docs/pages/blog/2023-discord-announcement.md

```diff
@@ -0,0 +1,55 @@
+---
+title: 'MUI is now on Discord!'
+description: Come join our community to engage in lively discussions, share your projects, and interact with the MUI team
+date: 2023-08-02T00:00:00.000Z
+authors: ['richbustos']
+tags: ['News']
+card: true
+---
+
+<a href="https://mui.com/r/discord"><img src="/static/blog/2023-discord-announcement/discord.png" width="1280" height="640" style="margin-bottom: 16px;" alt="Discord banner with link" /></a>
+
+Creating a place where we can interact with our community has been our priority for this year.
+Majority of our discussions and interactions with you have been through our [Twitter](https://twitter.com/MUI_hq), [GitHub](https://github.com/mui), and [marketing page](https://mui.com/).
+
+That's why we have officially launched our [Discord Server](https://mui.com/r/discord/) and would love for you to be part of our community!
+
+Come join to engage in lively discussions, share your projects, and interact with the MUI team & others!
```

Comment:
Avoid ampersands unless there's a character limit (Twitter); let's describe these "others" in more detail.

```suggestion
Come join to engage in lively discussions, share your projects, and interact with the MUI team as well as developers just like you from all around the world.
```